cEP-0024.md

```diff
@@ -0,0 +1,129 @@
+# Implement GitMate automations and plugins for coala
+
+| Metadata |                                                       |
+| -------- | ----------------------------------------------------- |
+| cEP      | 24                                                    |
+| Version  | 1.0                                                   |
+| Title    | Implement Gitmate automations and plugins for coala   |
+| Authors  | Vamshi Krishna Bommerla <mailto:bommvams72@gmail.com> |
+| Status   | Proposed                                              |
+| Type     | Process                                               |
+
+## Abstract
+
+This cEP describes the improvements to the coala workflow and the necessary
+automation to realise it, as a part of the
+[GSoC project](https://summerofcode.withgoogle.com/projects/#5344316227059712).
+
+## Introduction
+
+[GitMate](https://gitmate.io/) is an automation tool for developers. While it
+works well, there are a number of missing features which would save time of
+coala community, for example automatically close PRs opened on master of forks.
+Many of the plugins are general plugins, which are required by every
+organization, but coala needs more plugins which support coala development
+workflow. This project adds plugins to auto-review or auto-reject PRs,
+assign issues.
+
+## New Features
+
+### Auto-assign issue when it is mentioned in any of PR commits
+
+When a PR is opened and if any of it's commit message mentions (Fixes, Closes)
+any unassigned issues and there is no CI failure, auto
+assign PR author to the mentioned unassigned issues. Comment on the
+PR (configurable) if someone else is working on the mentioned issue. If PR
```

Comment:
what will the comment default be?

Or what do you believe the coala comment should be?

We need these details to check that the initial implementation will work nicely.

It also helps the team quickly see how it will work.